cmd/brew-test-bot.rb

```diff
@@ -530,7 +530,7 @@ def download
         unless @modified_formulae.empty?
           or_later_diff = Utils.popen_read(
             "git", "-C", @repository, "diff",
-            "-G    sha256 ['\"][a-f0-9]*['\"] => :\\w+_or_later$",
+            "-G    sha256 ['\"][a-f0-9]*['\"] => :\w+_or_later$",
```

Comment:
I don't get this. Doesn't this break it instead of fix it?
```
irb(main):001:0> "\w"
=> "w"
```

Reply:
```
irb(main):005:0> or_later.match(/sha256 ['\"][a-f0-9]*['\"] => :\\w+_or_later$/)
=> nil
irb(main):006:0> or_later.match(/sha256 ['\"][a-f0-9]*['\"] => :\w+_or_later$/)
=> #<MatchData "sha256 \"491d5a425463fb4f328503f91cfe6e28a0785fdbf8ec7a323a366bab54e4158c\" => :el_capitan_or_later">
```

Reply:
@reitermarkus https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/7753/version=sierra/console

If this isn't the cause, do you know what is?

Reply:
Bisected it to
```
b03e727b95e6401042a499d4b74782851599d2e5 is the first bad commit
commit b03e727b95e6401042a499d4b74782851599d2e5
Author: Mike McQuaid <mike@mikemcquaid.com>
Date:   Sat Nov 26 15:08:18 2016 +0000

    Don't build bottles by default on Travis CI. (#27)
    
    As they can't be securely uploaded anywhere for PRs.
    
    Also, set `--build-from-source` correctly when `--no-bottle` is passed.

:040000 040000 831db092c91d33296eb6ccb77177ef2216cec99b df07623a97517b13023a8a1b46d6a9c4c1cfac54 M	cmd
```